utils/rpms/daos.spec

```
@@ -134,9 +135,8 @@ Requires: hwloc
 Requires: mercury = %{mercury_version}
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
-Requires: libfabric >= 1.8.0
-%{?systemd_requires}
-Obsoletes: cart
+Requires: libfabric >= 1.11.0
+%systemd_requires
```

Looks like a merge conflict resolution here removed some lines:

```suggestion
%{?systemd_requires}
Obsoletes: cart
```